pkgs/by-name/ca/cargo-unfmt/package.nix

```diff
@@ -0,0 +1,35 @@
+{ lib
+, stdenv
+, fetchFromGitHub
+, rustPlatform
+, darwin
+, libiconv
+}:
+
+rustPlatform.buildRustPackage {
+  pname = "cargo-unfmt";
+  version = "0.3.3";
+
+  src = fetchFromGitHub {
+    owner = "fprasx";
+    repo = "cargo-unfmt";
+    rev = "0f4882f65d248e32812e0e854fa11d7db60921e7";
+    hash = "sha256-nvn4nZkkNQQvzShwoxtFqHeyhXQPm2GJoTKBI+MkFgM=";
+  };
+
+  cargoHash = "sha256-mMeHTYCUIZR3jVvTxfyH4I9wGfUdCWcyn9djnksAY8k=";
+
+  buildInputs = lib.optionals stdenv.isDarwin [ libiconv darwin.apple_sdk.frameworks.Security ];
+
+  doCheck = false;
+
+  outputs = [ "out" ];
+
+  meta = with lib; {
+    description = "Unformat code into perfect rectangles";
+    license = licenses.gpl3;
+    mainProgram = "cargo-unfmt";
+    maintainers = with maintainers; [ cafkafk ];
+    platforms = platforms.unix ++ platforms.windows;
```

Comment:
```suggestion
```

I'd recommend removing this line so every platform that has rust support can use it. Mentioning windows here looks weird.